lib/rules/no-invalid-position-declaration/README.md

```diff
@@ -0,0 +1,62 @@
+# no-invalid-position-declaration
+
+Disallow invalid position declarations.
```

Comment:
nitpick

It's not a big deal but it doesn't sound right to me.
Maybe it could be rephrased somehow?